config.example/group_vars/slurm-cluster.yml

```diff
@@ -99,8 +99,8 @@ allow_user_set_gpu_clocks: no
 ################################################################################
 # Enroot & Pyxis                                                               #
 ################################################################################
-slurm_install_enroot: true
-slurm_install_pyxis: true
+slurm_install_enroot: false 
```

Comment:
I think we want these by default... we should check and not run the playbook/role elsewhere or add it to the role

Reply:
Yeah agreed. I opened a PR on the enroot role to add CentOS support: https://github.com/NVIDIA/ansible-role-enroot/pull/6